cmd/root.go

```diff
@@ -85,5 +86,8 @@ func initConfig() {
 	// If a config file is found, read it in.
 	if err := viper.ReadInConfig(); err == nil {
```

Comment:
Reading config with viper was setup out of the box with cobra, but I didn't know about it so I wrote my own config readers. This could probably be removed entirely, but at some point I might switch over: https://github.com/plentico/plenti/issues/60